docs/shared-cluster.md

```diff
@@ -37,32 +37,25 @@ export KUBECONFIG=$PWD/secrets/shared-cluster.kubeconfig
 
 ## Creating / Deleting the Shared Cluster
 
-The shared cluster has the following attributes:
-
-* Tenant: AME
-
-* Subscription: ARO CI/E2E
-
-* Region: Westcentralus
-
-* Name: shared-cluster
-
-* Resource group name: shared-cluster
-
-* Cluster resource group name: aro-shared-cluster
```

Comment:
It might be good to replace these details with generalized details about where we deploy the shared cluster to.